gtsfm/common/gtsfm_data.py

```diff
@@ -127,7 +130,11 @@ def add_track(self, track: SfmTrack) -> bool:
         self._tracks.append(track)
         return True
 
-    def add_camera(self, index: int, camera: PinholeCameraCal3Bundler,) -> None:
+    def add_camera(
+        self,
+        index: int,
+        camera: PinholeCameraCal3Bundler,
+    ) -> None:
```

Comment:
did you reformat with 80 line limit?

Reply:
i did, strange but must be >120

Reply:
i took out the comma and seems to be fixed now